scientist-softserv/harvard-curiosity
# Revert rails spotlight

** Revert Rails to 6.1.5 and Spotlight to 3.3.0 and lock**
* * *

# What does this Pull Request do?
We were seeing some weird behavior with Rails 6.1.5 and with spotlight 3.5 so we are reverting to unblock other PRs and to investigate further

# How should this be tested?

Spin up locally in production mode -- make sure you can access exhibits and that things work. Also try running a bin/rails db:migrate in the container to see that rails isn't throwing warnings.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
